cloudmarker/clouds/gcpcloud.py

```diff
@@ -26,21 +26,22 @@
 class GCPCloud:
     """GCP cloud plugin."""
 
-    def __init__(self, key_file_path):
+    def __init__(self, key_file_path, _max_projects=0):
         """Create an instance of :class:`GCPCloud` plugin.
 
+         Note: The ``_max_projects`` argument should be
+         used only in the development-test-debug phase. It should not
+         be used in production environment. This is why we use the
+         convention of beginning it's name with underscore.
```

Comment:
Each line should try to occupy as much of the 72 character soft-limit as possible without exceeding 72 characters. The hard-limit is 79 characters. In this case, the first line is much shorter (55 characters only when it could be longer).

The wrapping of lines should not be done manually. It is quite hard to get this right manually, especially, when one wants to update the lines again after formatting. You should try to figure out if your editor can help you do this. With Vim I enter `:set textwidth=72` (it's set in my `~/.vimrc`) and then use the normal mode commands `gqap` or `gwap` to format the current paragraph or line correctly into multiple lines. If you are using another editor, you would have to figure out a way to automate this, perhaps with additional plugins.